cumulus_library/studies/core/core_templates/documentreference.sql.jinja

```diff
@@ -22,7 +22,6 @@ WITH temp_documentreference AS (
                     'docstatus',
                     'context',                
                     ('subject', 'reference', 'subject_ref'),
-                    ('context', 'encounter', 'encounter_ref'),
```

Comment:
Because context.encounter was not a requested field from the schema, this would always result in NULL. And it's an array anyway, so I don't think this is the best access approach. I switched to an unnest below.